lib/libvarnish/vbt.c

```diff
@@ -157,3 +157,19 @@ VBT_format(struct vsb *vsb)
 	vbt_execinfo(vsb);
 #endif
 }
+
+int
+VBT_dump(size_t len, char buf[len])
+{
+	struct vsb vsb[1];
+
+	if (buf == NULL || VSB_init(vsb, buf, len) == NULL)
+		return (-1);
+
+	VSB_printf(vsb, "Backtrace:\n");
+	VSB_indent(vsb, 2);
+	VBT_format(vsb);
+	VSB_indent(vsb, -2);
```

Comment:
The unindent statement is a no-op.

Reply:
That's true, but it was kept to maintain symmetry here. If you want I can remove it.